base/cvd/build_external/gfxstream/PATCH.gfxstream.3683847.patch

```diff
@@ -0,0 +1,836 @@
+commit 3683847ab3d60808a606d29843ad26840c8b32d6
```

Comment:
I see this commit is upstream here:

https://github.com/google/gfxstream/commit/3683847ab3d60808a606d29843ad26840c8b32d6

Is it possible to uprev the gfxstream commit hash in `base/cvd/MODULE.bazel` rather than pull in this individual commit?

Reply:
It is, but I pinged Jason about this option and we agreed to do it this way, so that we don't have to preempt the validation of a new gfxstream, and we don't have to do more work on either side just for the sake of warning fixes. When upstream comes in, the patches can just be dropped.